servers/firecone/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "firecone",
+    "name": "Firecone.eu",
+    "description": "survival events and more",
+    "addresses": [
+        "mc.firecone.eu"
+    ],
+    "primaryAddress": "mc.firecone.eu",
+    "minecraftVersions": [
+        "1.8",
+        "1.9",
+        "1.18",
```

Comment:
We've updated the way we handle minecraftVersions. Please check the README.MD for more info.

Reply:
Like this?